cypress/integration/regressions.spec.js

```diff
@@ -99,15 +99,6 @@ context('Regressions', () => {
       .should('contain', 'atâhk')
   })
 
-  // https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/176
-  describe('should show idiomatic lemmas', () => {
-    it('The Cree word ayiwinis should give you ayiwinisa as lemma', () => {
```

Comment:
the new hfstol gives "ayiwin" as the lemma, as the linguists decide "ayiwinisa" is not idiomatic enough.